core/src/wallet/bitcoin/observers/BitcoinLikeBlockchainObserver.cpp

```diff
@@ -50,26 +50,26 @@ namespace ledger {
         void BitcoinLikeBlockchainObserver::putTransaction(const BitcoinLikeBlockchainExplorerTransaction &tx) {
             std::lock_guard<std::mutex> lock(_lock);
             for (const auto& account : _accounts) {
-                account->run([account, tx] () {
-                    soci::session sql(account->getWallet()->getDatabase()->getPool());
-                    if (account->putTransaction(sql, tx) != BitcoinLikeAccount::FLAG_TRANSACTION_IGNORED)
-                        account->emitEventsNow();
-                });
+                account->run([account, tx]() {
+                                 BitcoinBlockchainObserver::emitEvent(account, [tx](soci::session &sql,
```

Comment:
The indentation is very big here. Can you reduce it, please?

Reply:
I'm all for backporting the `.clang-format` and having a big bang commit on `develop` to format the branch